src/lib/containers/markdown/MarkdownSynapse.tsx

```diff
@@ -179,63 +180,7 @@ export default class MarkdownSynapse extends React.Component<
     const initText = this.props.renderInline
       ? this.state.md.renderInline(markdown)
       : this.state.md.render(markdown)
-    const cleanText = sanitizeHtml(initText, {
-      allowedAttributes: {
-        a: ['href', 'target'],
-        button: ['class'],
-        div: ['class'], // PORTALS-1450: including 'style' in the allow-list will cause string values to come through, which crashes the app when used (because it uses jsx).
-        h1: ['toc'],
-        h2: ['toc'],
-        h3: ['toc'],
-        h4: ['toc'],
-        h5: ['toc'],
-        h6: ['toc'],
-        li: ['class'],
-        ol: ['class'],
-        span: ['*'],
```

Comment:
I confirmed that event handlers that could run scripts (e.g. `onclick`) were removed, but this `*` still makes me deeply uncomfortable.